homeassistant/components/flume/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "domain": "flume",
+    "name": "Flume",
+    "documentation": "https://www.home-assistant.io/integrations/flume/",
+    "dependencies": [],
+    "codeowners": ["@ChrisMandich"],
+    "requirements": ["pytz==2019.2","pyflume==0.1.4"]
```

Comment:
Nothing huge but I noticed most of the manifest.json are getting this order : 
domain
name
documentation
requirements
dependencies
codeowners

Can you put it the same way ?
And open requirements in multiple lines.

Reply:
Updated.